port_ocean/core/handlers/webhook/abstract_webhook_processor.py

```diff
@@ -1,12 +1,14 @@
 from abc import ABC, abstractmethod
 from loguru import logger
 
+from port_ocean.core.integrations.mixins.live_events import LiveEventsMixin
+from port_ocean.core.ocean_types import RAW_ITEM
 from port_ocean.exceptions.webhook_processor import RetryableError
 
 from .webhook_event import WebhookEvent, EventPayload, EventHeaders
 
 
-class AbstractWebhookProcessor(ABC):
+class AbstractWebhookProcessor(ABC, LiveEventsMixin):
```

Comment:
there seems to be some inconsistency between `webhook` and `liveEvents` what is the difference in terminology?